src/openvic-dataloader/detail/InternalConcepts.hpp

```diff
@@ -6,19 +6,23 @@
 #include <openvic-dataloader/NodeLocation.hpp>
 #include <openvic-dataloader/detail/Encoding.hpp>
 #include <openvic-dataloader/detail/SymbolIntern.hpp>
+#include <openvic-dataloader/detail/Utility.hpp>
 
 #include <lexy/encoding.hpp>
 #include <lexy/input/buffer.hpp>
 
+#include <dryad/symbol.hpp>
+
 #include <fmt/core.h>
 
 #include <lexy_ext/report_error.hpp>
 
 namespace ovdl::detail {
 	template<typename T>
 	concept IsFile =
-		requires(T t, const typename T::node_type* node, NodeLocation location) {
+		requires(T t, const T ct, const typename T::node_type* node, NodeLocation location) {
 			typename T::node_type;
+			{ ct.size() } -> std::same_as<size_t>;
```

Comment:
Out of interest, is there any difference/advantage to using `T t, const T ct` vs `T& t, const &T ct` here?

Reply:
no, ref calls are only able to be a subset of non-ref calls